package.json

```diff
@@ -47,6 +47,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.2.1",
     "eslint-plugin-standard": "^4.0.1",
+    "eslint-plugin-unicorn": "^46.0.1",
```

Comment:
Could this be pushed down into `spec/package.json`? Or are there plans to also make use of it in non-test code? Trying to do a better job of keeping the top-level `package.json` clean of test-specific packages.

Reply:
I would like to apply this outside of spec as well in follow up PRs